resources/help_entries/diseases.py

```diff
@@ -0,0 +1,45 @@
+#
+#
+# Custom Help entries for diseases
+#
+#
+
+# HEADER
+
+from os import stat
+from resources.help_entries.birthsigns import HELP_CATEGORY
+from evennia import create_help_entry
+from evennia.help.models import HelpEntry
+from evennia.utils import dedent
+from world.diseases import ALL_DISEASES
+from resources.help_entries.help_entry_wrapper import HelpEntryWrapper
+
+HELP_CATEGORY = 'disease'
+
+
+class RaceSignHelpEntry(HelpEntryWrapper):
+    __help_category__ = HELP_CATEGORY
+
+
+# CODE
+
+diseases = ", ".join([x.name.capitalize() for x in ALL_DISEASES])
```

Comment:
In order to get the name of all diseases you need to have a way of actually getting it. 

For signs (which I see you copy/pasted from) the list of all signs is initialized which is why I you can access the property of `name` 

Diseases doesn't have that (not yet anyways) so you could access class name variable like `__name__` such as 

```python
diseases = [x.__name__ for x in ALL_DISEASES]
```